cypress/integration/save.cy.js

```diff
@@ -62,18 +62,19 @@ const TEST_INDICATOR_NAMES = TEST_INDICATORS.slice(1, 4).map(
 
 describe('saving an AO', () => {
     describe('"save" and "save as" for a new AO', () => {
-        it('navigates to the start page', () => {
+        it('"save" and "save as" for a new AO', () => {
+            // navigates to the start page
             goToStartPage()
-        })
-        it('checks that Save is disabled', () => {
+
+            // checks that Save is disabled
             clickMenuBarFileButton()
             expectFileMenuButtonToBeDisabled(FILE_MENU_BUTTON_SAVE_EXISTING)
             closeFileMenuWithClick()
-        })
-        it(`changes vis type to ${TEST_VIS_TYPE_NAME}`, () => {
+
+            // changes vis type to ${TEST_VIS_TYPE_NAME}
```

Comment:
the const in the comment doesn't make much sense, maybe add a cypress.log instead to make it clear what `TEST_VIS_TYPE_NAME` is for the current run (as was previously made in the `it`.

e.g.
```
// changes vis type
cy.log(`changes vis type to ${TEST_VIS_TYPE_NAME}`)
```